src/components/atoms/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { StyledTextArea } from './TextArea.styled';
+
+export interface IProps {
+  className?: string;
```

Comment:
I think we can add placeholder as optional value and as a default you can put this `Type something...` content